src/components/blank/style.scss

```diff
@@ -0,0 +1,27 @@
+//------------------------------
+// *****************************
+// Tag Blank
+// *****************************
+//------------------------------
+
+@mixin prefix($property, $value, $prefixes: ()) {
+  @each $prefix in $prefixes {
+    -#{$prefix}-#{$property}: $value;
+  }
+
+  #{$property}: $value;
+}
+
+.tag-blank {
+  @include prefix(transform, translate(-50%, -50%), 'webkit' 'ms');
```

Comment:
Can't this be handled by `autoprefixer`?

Reply:
its temporary, will remove it (just trying some stuff out)